hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -744,17 +744,18 @@ enum SnapshotStatusProto {
  * SnapshotInfo table entry for Bucket/Volume snapshot metadata
  */
 message SnapshotInfo {
-  required string snapshotID = 1;
-  required string name = 2;
-  required string volumeName = 3;
-  required string bucketName = 4;
-  required SnapshotStatusProto snapshotStatus = 5;
-  required uint64 creationTime = 6;
-  required uint64 deletionTime = 7;
-  required string pathPreviousSnapshotID = 8;
-  required string globalPreviousSnapshotID = 9;
-  required string snapshotPath = 10;
-  required string checkpointDir = 11;
+  optional string snapshotID = 1;
+  optional string name = 2;
+  optional string volumeName = 3;
+  optional string bucketName = 4;
+  optional SnapshotStatusProto snapshotStatus = 5;
+  optional uint64 creationTime = 6;
+  optional uint64 deletionTime = 7;
+  optional string pathPreviousSnapshotID = 8;
+  optional string globalPreviousSnapshotID = 9;
+  optional string snapshotPath = 10;
+  optional string checkpointDir = 11;
+  optional int64 dbTxSequenceNumber = 12;
```

Comment:
To whoever reviewing this: the rationale behind **always** using `optional` protobuf fields from now on is that proto3 has already ditched `required`, meaning **all** proto3 fields are `optional`.

More reading: https://stackoverflow.com/a/52360213

Though we are still using proto2 in this case, using `optional` should make the future conversion to proto3 easier (if that day ever comes).

Also, message `SnapshotInfo` is newly added in the snapshot dev branch and not merged to master branch yet. So no compatiblity concerns here.

Reply:
thanks siyao for the reminder!